x-pack/plugins/observability_solution/observability_onboarding/public/application/quickstart_flows/otel_kubernetes/otel_kubernetes_panel.tsx

```diff
@@ -213,17 +225,33 @@ helm install opentelemetry-kube-stack open-telemetry/opentelemetry-kube-stack \\
                   ]}
                 />
                 <EuiSpacer />
-                <EuiCodeBlock paddingSize="m" language="yaml" lineNumbers={{ highlight: '5-6' }}>
-                  {`apiVersion: v1
-kind: Pod
+                <EuiCodeBlock paddingSize="m" language="yaml">
+                  {`# To annotate specific deployment Pods modify its manifest
+apiVersion: apps/v1
+kind: Deployment
 metadata:
-  name: my-app
-  annotations:
-    instrumentation.opentelemetry.io/inject-${idSelected}: "${namespace}/elastic-instrumentation"
+  name: myapp
 spec:
-  containers:
-  - name: my-app
-    image: my-app:latest`}
+  ...
+  template:
+    metadata:
+      annotations:
+        instrumentation.opentelemetry.io/inject-${idSelected}: "${namespace}/elastic-instrumentation"
+      ...
+    spec:
+      containers:
+      - image: myapplication-image
+        name: app
+      ...
+
+# To annotate all resources in a namespace
+kubectl annotate namespace my-namespace instrumentation.opentelemetry.io/inject-${idSelected}: "${namespace}/elastic-instrumentation"
+
+# Restart your deployment
+kubectl rollout restart deployment myapp -n my-namespace
+
+# Check annotations have been applied correctly
+kubectl describe deployment myapp -n my-namespace`}
```

Comment:
I would recommend to follow the latest suggestion by @gizas to ask users to describe Pods and not the deployment (I've explained the reasons at https://github.com/elastic/kibana/issues/197620#issuecomment-2445229752).